### DISCUSSION TITLE
Vst's not showing up in "+" pulldown

### DISCUSSION THREAD

**Discussion by GroverWayne** [apohl79/audiogridder#1303] [General]
I've installed Audiogridder on my PC (Win 10 x64) and I'm using Cubase 9 Pro. Audiogridder installed fine, however, when I go to use it in a session, it loads fine, but there's no vst's to be found in the "+" dropdown. 
Help Please??

**Comment by apohl79** [comment]
The AG plugin shows plugins AG server has scanned successfully. You can check the available plugins on the server side via the plugin manager that you can open through the tray menu.

**Comment by GroverWayne** [comment]
Thx for the speedy reply, I can’t see plug-ins any where on AG, sever or master. Both computers have conected but no Vst’s. It’s so weird. Can’t wait to figure this out and see how it works. Sent from my iPhoneOn Dec 17, 2023, at 3:44 PM, Andreas Pohl ***@***.***> wrote:﻿
The AG plugin shows plugins AG server has scanned successfully. You can check the available plugins on the server side via the plugin manager that you can open through the tray menu.

**Comment by GroverWayne** [comment]
Sent from my iPhoneOn Dec 17, 2023, at 6:03 PM, Aaron Reeves ***@***.***> wrote:﻿<image0.jpeg><image1.jpeg>Sent from my iPhoneOn Dec 17, 2023, at 6:01 PM, Aaron Reeves ***@***.***> wrote:﻿Thx for the speedy reply, I can’t see plug-ins any where on AG, sever or master. Both computers have conected but no Vst’s. It’s so weird. Can’t wait to figure this out and see how it works. Sent from my iPhoneOn Dec 17, 2023, at 3:44 PM, Andreas Pohl ***@***.***> wrote:﻿
The AG plugin shows plugins AG server has scanned successfully. You can check the available plugins on the server side via the plugin manager that you can open through the tray menu.

**Comment by GroverWayne** [comment]
I finally figured out what was wrong and both computers are linked successfully I can get about 1/20 of my VST to install. The rest just say failed in red. None of my instruments will install somehow it can’t read the licensing and says, reinstall the programs. Is this a common thing? Is there a way around this? Am I not sharing the licensing files somehow?Sent from my iPhoneOn Dec 17, 2023, at 6:01 PM, Aaron Reeves ***@***.***> wrote:﻿Thx for the speedy reply, I can’t see plug-ins any where on AG, sever or master. Both computers have conected but no Vst’s. It’s so weird. Can’t wait to figure this out and see how it works. Sent from my iPhoneOn Dec 17, 2023, at 3:44 PM, Andreas Pohl ***@***.***> wrote:﻿
The AG plugin shows plugins AG server has scanned successfully. You can check the available plugins on the server side via the plugin manager that you can open through the tray menu.
